.env.example

```diff
@@ -0,0 +1,26 @@
+####### API KEYS #######
```

Comment:
Can you clearly mark which ones are `OPTIONAL`. For example we definitely need `REACT_APP_ALCHEMY_IDS` but INFURA_IDS and REACT_APP_SELF_HOSTED_WEBSOCKETS doesn't have to exist.

Reply:
literally none of them are rquired in development environment

Reply:
added commentary in. https://github.com/OlympusDAO/olympus-frontend/pull/415/commits/536f552bb114551f82ac711dd82a77355986246f